packages/report-server/src/reportBuilder/context/buildContext.ts

```diff
@@ -34,8 +34,13 @@ const buildOrgUnits = async (reqContext: ReqContext, data: FetchResponse) => {
   );
 };
 
+const buildDataElementCodeToName = async (reqContext: ReqContext, data: FetchResponse) => {
+  return data.metadata?.dataElementCodeToName || {};
```

Comment:
Right now this `metadata` is throwing a typescript error since it's not expected to be on the `FetchResponse` type. That's about to change as of: https://github.com/beyondessential/tupaia/pull/3299 so I'm happy to leave this as an error for now and then when that gets merged in it will go away 👍